### PR TITLE
implemented HW_28.2: removed all static variables user_id from code

### PR DESCRIPTION
**Pull request by MaliukDaria**

### CODE REVIEW

**Review by DmGrt — APPROVED** [review]
Good job!

**Review by Victoria-Moiseienko — COMMENTED** [review]
Good job :)

File: src/main/java/com/online/store/controllers/shoppingcart/controller/DeleteProductFromShoppingCartController.java [hunk 1]
```diff
@@ -21,7 +21,9 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws IOException {
         Long productId = Long.valueOf(req.getParameter("id"));
         shoppingCartService.deleteProduct(
-                shoppingCartService.getByUserId(USER_ID), productService.getById(productId));
+                shoppingCartService.getByUserId(
+                        (Long) req.getSession().getAttribute(USER_ID)),
+                productService.getById(productId));
```
Comment (KseniiaMakarova) [inline src/main/java/com/online/store/controllers/shoppingcart/controller/DeleteProductFromShoppingCartController.java]: local variables would make the code more readable here

**Review by KseniiaMakarova — APPROVED** [review]

File: src/main/java/com/online/store/controllers/shoppingcart/controller/DeleteProductFromShoppingCartController.java (hunk 1 above)
Comment (MaliukDaria) [reply src/main/java/com/online/store/controllers/shoppingcart/controller/DeleteProductFromShoppingCartController.java]: done)

Comment (MaliukDaria) [reply src/main/java/com/online/store/controllers/shoppingcart/controller/DeleteProductFromShoppingCartController.java]: done

**Pull request merged**